### PR TITLE
chore(condo): INFRA-674 codegen speed boost 🚀

### PR DESCRIPTION
**Pull request by Alllex202**
Executing `eslint --fix` takes a very loooooong time.
Removed it from codegen

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ module.exports = {
     ignorePatterns: [
         'dist',
         '*schema.ts',
+        ...['condo', 'callcenter'].map(app => `apps/${app}/gql/*.ts`),
```
Comment (sitozzz) [inline .eslintrc.js:38]: Why only 2 of apps, not all of them?

Comment (Alllex202) [reply .eslintrc.js:38]: You are right, it is better to add it everywhere at once

**Comment by Alllex202** [comment]
> Is that an expected behavior? I can't find any changes at the linter side but there are a lot of trailing commas everywhere

linter was turn off for codegen because run very long time

**Review by sitozzz — APPROVED** [review]
Okay, but you should resolve conflicts first)

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**